classes/help.php

```diff
@@ -155,32 +154,32 @@ function print_page() {
+														<?php printf( __( '%s Auto-load search results:%s The locations will load based on the default location, default search radius and zoom level you have set.', 'simplemap' ), '<strong>', '</strong>' ); ?>
 														<br/>
-														<?php printf( __( '%s Auto-load all locations:%s All of the locations in your database will load at the default zoom level you have set, disregarding your default search radius. %s This option is not enabled if you have more than 100 locations in your database.%s', 'SimpleMap' ), '<strong>', '</strong>', '<em>', '</em>' ); ?>
+														<?php printf( __( '%s Auto-load all locations:%s All of the locations in your database will load at the default zoom level you have set, disregarding your default search radius. %s This option is not enabled if you have more than 100 locations in your database.%s', 'simplemap' ), '<strong>', '</strong>', '<em>', '</em>' ); ?>
 														<br/><br/>
 
-														<?php _e( 'If you leave the checkbox unchecked, then the auto-load feature will automatically move the map to the center of all the loaded locations. If you check the box, your default location will be respected regardless of the locations the map is loading.', 'SimpleMap' ); // XSS ok. ?>
+														<?php _e( 'If you leave the checkbox unchecked, then the auto-load feature will automatically move the map to the center of all the loaded locations. If you check the box, your default location will be respected regardless of the locations the map is loading.', 'simplemap' ); // XSS ok. ?>
 													</td>
 												</tr>
 
 												<tr valign="top">
 													<td width="150">
-														<strong><?php _e( 'Special Location Label', 'SimpleMap' ); // XSS ok. ?></strong>
+														<strong><?php _e( 'Special Location Label', 'simplemap' ); // XSS ok. ?></strong>
 													</td>
-													<td><?php _e( 'This is meant to flag certain locations with a specific label. It shows up in the search results with a gold star next to it. Originally this was developed for an organization that wanted to highlight people that had been members for more than ten years. It could be used for something like that, or for "Favorite Spots," or "Free Wi-Fi," or anything you want. You can also leave it blank to disable it.', 'SimpleMap' ); // XSS ok. ?></td>
+													<td><?php _e( 'This is meant to flag certain locations with a specific label. It shows up in the search results with a gold star next to it. Originally this was developed for an organization that wanted to highlight people that had been members for more than ten years. It could be used for something like that, or for "Favorite Spots," or "Free Wi-Fi," or anything you want. You can also leave it blank to disable it.', 'simplemap' ); // XSS ok. ?></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 506 characters

<!-- Comment by [SideCI](https://sideci.com) -->